index.md

```diff
@@ -127,8 +127,11 @@ Thinking about using Anbox Cloud for your next project? [Get in touch!](https://
 | 3 | howto/cluster/scale-down | [Scale down a LXD cluster](https://discourse.ubuntu.com/t/scale-down-a-lxd-cluster/24323)|
 | 2 | howto/anbox/landing | [Work with the Anbox runtime](https://discourse.ubuntu.com/t/how-to-work-with-the-anbox-runtime/33098) |
 | 3 | howto/anbox/develop-platform | [Develop a platform plugin](https://discourse.ubuntu.com/t/how-to-develop-a-platform-plugin/33099) |
-| 2 | howto/troubleshoot/landing | [Troubleshoot Anbox Cloud](https://discourse.ubuntu.com/t/anbox-cloud-faq/17837)|
-| 3 | howto/troubleshoot/collect-info | [Collect information](https://discourse.ubuntu.com/t/how-to-collect-troubleshooting-information/32263)|
+| 2 | howto/troubleshoot/landing | [Troubleshoot Anbox Cloud](https://discourse.ubuntu.com/t/how-to-troubleshoot-anbox-cloud/17837)|
+| 3 | howto/troubleshoot/troubleshoot-initial-setup | [Troubleshoot initial setup](tbd)|
+| 3 | howto/troubleshoot/troubleshoot-application-creation | [Troubleshoot application creation](tbd)|
+| 3 | howto/troubleshoot/troubleshoot-container-failures | [Troubleshoot container failures](tbd)|
+| 3 | howto/troubleshoot/troubleshoot-lxd-cluster | [Troubleshoot LXD cluster](tbd)|
```

Comment:
I was a bit skeptical to remove troubleshoot to avoid ambiguity but since it is just the file name and also it is embedded within the troubleshoot folder, it should be fine. Having shorter titles translating into URLs is the higher priority here.